.github/workflows/build_runson_ami.yaml

```diff
@@ -10,17 +10,18 @@ jobs:
   build_ami:
     name: Build-AMI
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
     steps:
       - name: Check out code
         uses: actions/checkout@v4
-        with:
-          # TODO: Rename this PAT to something that encompasses more than just cherry-picking
-          # but all PR creation (such as this one).
-          token: ${{ secrets.WORKER_PANTS_CHERRY_PICK_PAT }}
       - name: Run packer
         env:
-            GH_TOKEN: ${{ secrets.WORKER_PANTS_CHERRY_PICK_PAT }}
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          set -x +e
```

Comment:
The `+e`  was in my pyenv plugin since there were `if` statements checking exit status. Maybe not needed in this script?

Reply:
Ah whoops, I copy-pasted but intended `-e`.

Reply:
Also, `-x` would expose GITHUB_TOKEN, so I selectively turn it off around that part.